mediator/med_phases_aofluxes_mod.F90

```diff
@@ -310,12 +355,27 @@ subroutine med_aofluxes_init(gcomp, aoflux, FBAtm, FBOcn, FBFrac, FBMed_aoflux,
 
     call FB_GetFldPtr(FBAtm, fldname='Sa_z', fldptr1=aoflux%zbot, rc=rc)
     if (chkerr(rc,__LINE__,u_FILE_u)) return
-    call FB_GetFldPtr(FBAtm, fldname='Sa_u', fldptr1=aoflux%ubot, rc=rc)
-    if (chkerr(rc,__LINE__,u_FILE_u)) return
-    call FB_GetFldPtr(FBAtm, fldname='Sa_v', fldptr1=aoflux%vbot, rc=rc)
-    if (chkerr(rc,__LINE__,u_FILE_u)) return
-    call FB_GetFldPtr(FBAtm, fldname='Sa_tbot', fldptr1=aoflux%tbot, rc=rc)
-    if (chkerr(rc,__LINE__,u_FILE_u)) return
+
+    ! bulk formula quantities for nems_orig_data
+    if (trim(coupling_mode) == 'nems_orig_data' .and. ocn_surface_flux_scheme == -1) then
+       call FB_GetFldPtr(FBAtm, fldname='Sa_u10m', fldptr1=aoflux%ubot, rc=rc)
```

Comment:
So we are adding an option, when ocn_surface_flux_scheme == -1, the bulk formula will use the 10m wind and 2m tmp? Have we verified that model runs stably with this method in godas run experiment?

Reply:
I do think we need verification from the godas team that it is implemented correctly and running stably.